mailer.rst

```diff
@@ -479,7 +479,7 @@ several popular libraries):
 .. code-block:: terminal
 
     # instead of league/commonmark, you can also use erusev/parsedown or michelf/php-markdown
-    $ composer require twig/markdown-extension league/commonmark
+    $ composer require twig/markdown-extra league/commonmark
```

Comment:
Nice change! However, we need to update other contents of this file. The new extension adds a filter called `markdown_to_html` instead of `markdown` so we also need to update the code example a bit.

Reply:
@javiereguiluz you're right, i totally forgot about them. done 😉